cinema/views.py

```diff
@@ -34,6 +37,26 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(parameters):
+        return [int(str_id) for str_id in parameters.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset.prefetch_related("actors", "genres")
+
+        if actors := self.request.query_params.get("actors", None):
+            actors_id = self._params_to_ints(actors)
+            queryset = queryset.filter(actors__id__in=actors_id)
+
+        if genres := self.request.query_params.get("genres", None):
+            genres_id = self._params_to_ints(genres)
+            queryset = queryset.filter(genres__id__in=genres_id)
+
+        if title := self.request.query_params.get("title", None):
+            queryset = queryset.filter(title__icontains=title)
```

Comment:
None is a default second parameter for .get() method, so you don't need to add it